source/facets.toml

```diff
@@ -0,0 +1,7 @@
+[[facets]]
+category = "target_product"
+value = "drivers"
+
+[[facets]]
+category = "programming_language"
+value = "kotlin"
```

Comment:
This repo contains the landing pages for all of our drivers. I think this should be removed (or, if possible and useful, add a facet for every programming language on the page).

Reply:
Thanks for the background info, @mongoKart.  I agree; let's close out this PR, and we'll need to take the landing pages individually by hand.  The script doesn't seem like a good fit.